tests/test_ceo_direct_dedup.py

```
@@ -0,0 +1,193 @@
+import pytest
+
+ROOT = Path(__file__).resolve().parent.parent
+if str(ROOT) not in sys.path:
+    sys.path.insert(0, str(ROOT))
+
+from tasks.models import Task, TaskStatus
+from tasks.store import TaskStore
+from tasks.issue_intake import create_task_from_oldest_open_issue, issue_source_id
+
+
+def _make_issue(number: int, title: str = "Test issue", labels: list = None, body: str = "Test body"):
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)